src/app/bugs/components/bugs-technology/bugs-technology.component.ts

```diff
@@ -0,0 +1,29 @@
+import { Component, OnInit } from '@angular/core';
+import { chartsData, pieChartsData,  pieChartOPtion } from 'src/app/commons/constants/charts.constant';
+import { HttpService } from 'src/app/commons/services/http.service';
+import { SeverityModel } from '../../models/classes/severity.models.class';
+import { BugsGenericService } from '../../services/bugs-generic.service';
+import { BugsMapperService } from '../../services/bugs-mapper.service';
+
+@Component({
+  selector: 'app-bugs-technology',
+  templateUrl: './bugs-technology.component.html',
+  styleUrls: ['./bugs-technology.component.scss']
+})
+export class BugsTechnologyComponent implements OnInit {
+  public data = pieChartsData;
+  public options = pieChartOPtion;
+  public type= "pie";
```

Comment:
Shall we remove this property and use directly the value in html binding?
something as shown below:
<app-charts  class="box-details" chartType="pie" [chartOptions]="options" [chartData]="data"></app-charts>

Reply:
yes the type property removed and added as a binding in html